samples/nrf9160/http_update/application_update/README.rst

```diff
@@ -40,7 +40,7 @@ This image is automatically generated when building the application.
 Building and running
```

Comment:
I'm don't think the `samples.rst` will now list these samples given the readme files are nested deeper in the tree:
https://github.com/hakonfam/fw-nrfconnect-nrf-1/blame/http-update/doc/nrf/samples.rst#L37
This could probably be confirmed by some tech writer though.

Reply:
I will double check, thanks.

Reply:
fixed